### ISSUE TITLE
Where to find the session doc?

### ISSUE THREAD

**Issue by Vadorequest** [balderdashy/sails#2996]
It is sometimes really hard to find the right documentation.

For instance, I want to know how I can access the Sails Session but I can't find anything on the documentation:

http://sailsjs.org/#!/documentation/concepts

No "Session" link, not possible to type a keyword to find what I need...
This doc really needs a search input.

Thanks.

**Comment by Vadorequest** [comment]
Finally found it there: http://sailsjs.org/#!/documentation/reference/req

But still lacks a easy way to find resources.

**Comment by sailsbot** [comment]
Thanks for posting, @Vadorequest.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
